test_regress/driver.pl

```diff
@@ -1806,6 +1825,10 @@ sub _make_main {
         $set = "topp->";
     }
 
+    if ($self->{simbenchmark}) {
+        $fh->print("    const auto starttime = std::chrono::steady_clock::now();\n");
+    }
+
```

Comment:
I suggest start timing on the *second* time through the simulation main loop.  (Second so the caches warm up).

Reply:
As in the following?
```c++
    std::chrono::time_point<std::chrono::steady_clock> starttime;
    warm = false;
    while ((contextp->time() < sim_time * MAIN_TIME_MULTIPLIER)
           && !contextp->gotFinish()) {
        if(warm) {
            starttime = std::chrono::steady_clock::now();
        }
        topp->clk = !topp->clk;
        topp->eval();
        contextp->timeInc(1 * MAIN_TIME_MULTIPLIER);
        contextp->timeInc(1 * MAIN_TIME_MULTIPLIER);
        contextp->timeInc(1 * MAIN_TIME_MULTIPLIER);
        contextp->timeInc(1 * MAIN_TIME_MULTIPLIER);
        contextp->timeInc(1 * MAIN_TIME_MULTIPLIER);
        warm = true;
    }
```
Note that the `sim_time/exec_time` ratio is already a bit skewed from the `contextp->timeInc(10 * MAIN_TIME_MULTIPLIER);` call, before the main loop.

Reply:
Well, to be a bit cleaner

```
if (VL_UNLIKELY(warm)) {
   starttime = std::chrono::steady_clock::now();
   warm = true;
}
```

Perhaps the perf report should instead count ->eval calls (through a new counter you increment)?

Reply:
Counting eval's indeed seems like the better way to go - i'll take a look at it.